src/dmclock/test/CMakeLists.txt

```diff
@@ -31,5 +31,7 @@ endif()
   
 add_dependencies(dmclock-tests dmclock)
 
-add_test(NAME dmclock-tests
-  COMMAND $<TARGET_FILE:dmclock-tests>)
+if(NOT FREEBSD)
```

Comment:
I don't know all that building under FreeBSD needs to be able to do. But there are references to dmclock-tests in ceph:src/CMakeLists.txt, ceph:debian/ceph-test.install, and ceph:ceph.spec.in.